src/PostgREST/App.hs

```diff
@@ -206,7 +205,7 @@ app dbStructure conf apiRequest =
         else return notFound
 
     (ActionRead, TargetRoot, Nothing) -> do
-      body <- encode <$> H.query schema accessibleTables
```

Comment:
The standard way to request a certain format is with the Accept header, so in this case

```HTTP
Accept: application/swagger+json
```